languages/common-lisp/exercises/concept/sameness/sameness-test.lisp

```diff
@@ -0,0 +1,149 @@
+;; Ensures that sameness.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  ;; (load "sameness")
+  (load "./.meta/example.lisp")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from sameness and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage sameness-test
+  (:use :cl :fiveam :sameness)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :sameness-test)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite sameness-suite)
+(in-suite sameness-suite)
+
+(defparameter +an-array+ #(1 2 3))
+(defparameter +a-similar-but-different-array+ #(1 2 3))
```

Comment:
For the shorter items, like these arrays, I think it might look slightly cleaner to have them embedded in the maze list directly. It also might make the error messages from the test-runner a bit less confusing. But I think both approaches are good

Reply:
In one of the rooms it compare +an-array+ to +an-array+ and expects it to only work because they are `eq`. I'd need to use reader macro `#=` to do the same if i embed them into the rooms alist.

Reply:
That's a fair point! I'm happy with this then :)